content/en/docs/getting-started/_index.md

```diff
@@ -21,23 +21,55 @@ To run MicroShift, you need a machine with at least:
 
 For production deployments, we recommend (and only test) deploying MicroShift on RHEL 8, CentOS Stream 8, or Fedora 34+ using one of two methods:
 
-- running containerized on Podman
+- running containerized with `Podman`
 - installing via .rpm (e.g. for embedding MicroShift into an `rpm-ostree` image)
 
-Both methods feature a minimal resource footprint, a strong security posture, the ability to restart/update without disrupting workloads, and optionally auto-updates.
+### Install CRI-O
+
+MicroShift requires CRI-O to be installed and running on the host:
 
 {{< tabs >}}
-{{% tab name="Podman" %}}
-MicroShift requires CRI-O to be installed on the host:
+{{% tab name="RHEL" %}}
+
+```Bash
+command -v subscription-manager &> /dev/null \
+    && subscription-manager repos --enable rhocp-4.8-for-rhel-8-x86_64-rpms
+sudo dnf install -y cri-o cri-tools
+sudo systemctl enable crio --now
+```
+
+{{% /tab %}}
+{{% tab name="Fedora" %}}
 
 ```Bash
-command -v subscription-manager >/dev/null 2>&1 \
-    && subscription-manager repos --enable rhocp-4.8-for-rhel-8-x86_64-rpms \
-    || sudo dnf module enable -y cri-o:1.21
-sudo dnf install -y cri-o cri-tools podman
+sudo dnf module enable -y cri-o:1.21
+sudo dnf install -y cri-o cri-tools
+sudo systemctl enable crio --now
+```
+{{% /tab %}}
+{{% tab name="CentOS_8_Stream" %}}
+
+```Bash
+# OS=CentOS_8_Stream
```

Comment:
Should we remove the comments? (Or even substitute the vars immediately? dunno...) It's the only config we test and this way users can immediately copy&paste.

Reply:
sure, makes sense, will do that

Reply:
done